.travis.yml

```diff
@@ -2,13 +2,20 @@ sudo: false
 language: python
 python:
   - "3.6"
-install: pip install tox-travis
-script: tox
 
-deploy:
-  - provider: pypi
-    user: __token__
-    # password: see secret PYPI_PASSWORD variable
-    distributions: sdist bdist_wheel
-    on:
-      tags: true
+jobs:
+  include:
+    - name: test
+      install: pip install tox-travis
+      script: tox
+    - name: deploy
+      if: tag IS present
+      script: skip
```

Comment:
Oh, is `skip` some TravisCI trick?

Reply:
Yes: https://docs.travis-ci.com/user/build-stages/#build-stages-and-deployments
Though it doesn't matter here since there's nothing outside the `jobs:` section.... I just copied and pasted it from another config.

Reply:
Ah, thanks I didn't know about this and had various `echo dummy text` to solve that (because if left as a blank array, it doesn't override anything).